io/disk/port_bounce.py

```diff
@@ -146,25 +152,29 @@ def porttoggle(self, test_ports, sleep_time):
         except CommandFailed as cf:
             self.log.info("port disable failed for port(s) %s, details: %s",
                           test_ports, str(cf))
-        time.sleep(sleep_time)
         self.verify_port_disable(test_ports)
+        time.sleep(20)
         self.verify_port_toggle_host("Linkdown")
+        self.mpath_fail_check(test_ports)
 
+        time.sleep(sleep_time)
         # Port Enable
         self.log.info("Enable port(s) %s", test_ports)
         try:
             self.fc_run_command("portenable %s" % port_input)
         except CommandFailed as cf:
             self.log.info("port enable failed for port %s, details: %s",
                           test_ports, str(cf))
-        time.sleep(5)
+        time.sleep(20)
```

Comment:
I understand the need for waiting here. But there is no fool proof wait time. This is a good candidate for `avocado.utils.wait.wait_for`

Reply:
Also, consider moving the sleep / wait to corresponding functions instead of having here.